src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -102,6 +102,9 @@
   {:outcome (schema/enum :success :not-found :error)
    :message schema/Str})
 
+(def OIDMappings
+  {schema/Str schema/Keyword})
+
```

Comment:
It's pretty funky that this map is backwards from both the usual map style of keyword -> string as well as what `puppet-short-names` is already using.